renovate.json

```diff
@@ -1,10 +1,18 @@
 {
-  "ignoreDeps": ["dvc", "execa", "@types/vscode"],
+  "ignoreDeps": ["execa", "@types/vscode"],
   "extends": ["config:base"],
   "packageRules": [
+    {
+      "description": "Automatically merge non-major updates",
+      "automerge": true,
+      "automergeType": "branch",
```

Comment:
[F] This should create a branch for each update and run the CI, if the CI passes then the branch will be merged without a PR.

Reply:
There is the `vsce` dependency that is not covered by any checks. We once had problems publishing because it was merged (ok I merged it) without testing the publish.